.github/workflows/build-test.yml

```
@@ -0,0 +1,50 @@
+name: build-test
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)